draft-wing-cidfi.md

```diff
@@ -290,16 +345,33 @@ authorizes those network elements ({{client-authorizes}}).
 
 ## Client Learns Local Network Supports CIDFI {#discovery}
 
+For this step, four mechanisms are described: DNS SVCB records, IPv6
+Provisioning Domains (PvD), DHCP, or 3GPP PCO.  These are described
+below.
+
+> Standardizing on all or some of these mechanisms is for further discussion.
+
+In all cases below,
+
+- if the discovery succeeds, the client follows the processing in
+  {{client-authorizes}}.
+
+- if the discovery failed (i.e., the client concludes that the local
+  network does not support CIDFI), the processing stops.
```

Comment:
It is possible that the local network may not have CIDFI-aware network elements but the ISP may have CIDFI-aware network elements.  Replace "the client concludes that the local network does not support CIDFI" with "the client concludes the local network and ISP do not support CIDIF".